recipes/use-package

```diff
@@ -1,7 +1,13 @@
 (use-package
     :fetcher github
     :repo "jwiegley/use-package"
-    :files (:defaults (:exclude "bind-key.el"
-                                "bind-chord.el"
-                                "use-package-chords.el"
-                                "use-package-ensure-system-package.el")))
+    :files ("use-package*.el"
+            "dir"
+            "use-package*.info"
+            "use-package*.texi"
+            "LICENSE"
+            "*.md"
```

Comment:
Generally, `LICENSE` and `*.md` files etc are not included.

Default files spec is here:
https://github.com/melpa/melpa/blob/62271788f20919a741acb1e0477a823d300e55ee/package-build/package-build.el#L138-L143

Reply:
Thanks I'm aware, but @jwiegley requested that we add them anyway in case someone wants to cd into `.emacs.d/elpa` and read it before we finish the texinfo docs. Is that ok or do we need to stick with the convention of excluding `LICENSE` and `*.md`?

Reply:
If there's a standard of not including them, then let's just not include them; even more incentive to get the manual done.

Reply:
Removed.